src/modules/elements/tooltip/style.pcss

```diff
@@ -0,0 +1,14 @@
+.components-tooltip {
+
+	.components-popover__content {
+		white-space: normal;
+		min-width: 350px;
```

Comment:
Curious why we have a `min-width` here and below. If a user has content that is really short, we'll still have a tooltip that's `350px` wide (and a lot of space).

Reply:
So my point isn't really to use a different number, my question is rather why we have it.
- if we have short content, a min width will make the width to the min width value, which adds space. This seems suboptimal. 
- if we have long content, the min width doesn't matter. Only the max width does, which we have set.

Reply:
Oh, right. Yeah, I basically used that number because, if not, it's falling back to the number set here https://github.com/moderntribe/tribe-common/blob/f4fb8b981be2c0ed4740ead6a4b270450ac6cc45/src/modules/elements/time-picker/style.pcss#L56 and I'm not sure if that has a specific purpose.

Reply:
I think that's got another class attached to it, `.tribe-editor__timepicker__dialog`, so I don't think that should affect this tooltip popover, right?

Reply:
Gotcha. This is an issue with text-based width and absolutely-positioned items. In this case your max-width will never be used then (the width will always be 350px). Anyway, this should be fine.